data/human/ships.txt

```diff
@@ -3474,6 +3474,68 @@ ship "Surveillance Drone"
 
 
 
+ship "Valkyrie"
+	sprite "ship/pirate valkyrie"
+	thumbnail "thumbnail/pirate valkyrie"
+	attributes
+		category "Medium Warship"
+		"cost" 3900000
+		"shields" 4800
+		"hull" 2800
+		"required crew" 9
+		"bunks" 26
+		"mass" 157
+		"drag" 4.9
+		"heat dissipation" .63
+		"fuel capacity" 500
+		"cargo space" 41
+		"outfit space" 400
+		"weapon capacity" 170
+		"engine capacity" 76
+		weapon
+			"blast radius" 80
+			"shield damage" 800
+			"hull damage" 400
+			"hit force" 1200
+	outfits
+		"Quad Blaster Turret" 2
+		"Particle Cannon" 2
```

Comment:
```suggestion
```

Let's have this be a proper carrier that hangs back and pounds enemies with torpedoes while its fighters pin them down, rather than just having yet another frontline combatant-carrier that will give people a reason to pester us for a way to convert bays into something "more useful"?